LocalWiki.php

```diff
@@ -89,6 +89,18 @@ function onInactiveSiteNoticeAfter( &$siteNotice, $skin ) {
 	];
 }
 
+if ( $wmgWikiapiaryFooterPageName ) {
+ 	$wi->config->settings['+wgFooterIcons']['+default'] = [
```

Comment:
Remove +default, though I suppose this can become:

$wi->config->settings['+wgFooterIcons']['default']['poweredby']['wikiapiary']

Reply:
Oh yeah, completely true. My bad